src/controllers/selectserver.js

```diff
@@ -21,17 +23,24 @@ define(["loading", "appRouter", "layoutManager", "appSettings", "apphost", "focu
                 cardImageContainer = '<div class="cardImage" style="' + item.cardImageStyle + '"></div>';
             }
 
-            var cardBoxCssClass = "cardBox";
+            var cssClass = "card overflowSquareCard loginSquareCard scalableCard overflowSquareCard-scalable";
 
             if (layoutManager.tv) {
-                cardBoxCssClass += " cardBox-focustransform";
+
+                cssClass += ' show-focus'
+
+                if (enableFocusTransform) {
+                    cssClass += ' show-animation'
+                }
             }
```

Comment:
can this be extracted to a common function instead of being copy-pasted in like hundred places?

Reply:
I also thought about it, but the question is, where should it be?
Currently, this code placed next to the card generating code. I think that all card generating code should be in `cardBuilder` (need to export `buildCard` function), but I did not dare to do such refactoring.

Reply:
Yes, the main issue is that CardBuilder only really works with library items, but we use the same style in several other elements (login cards, server cards, library cards, device cards) without the component. We need a more generic Card element to use in those cases, but that's a huge change.

Reply:
Maybe adding a TODO comment in every location it gets copied so other people might notice in the future and make the refactor?

Reply:
For now, it is possible to extract focus related code into some function, and use it even with emby-button:
```
function focusClass() {

    var cssClass = ' show-focus';    // space because it must be added to the base class

    if (enableFocusTransform) {
        cssClass += ' show-animation';
    }

    return cssClass;
}
```
Or even
```
function focusClass() {

    var cssClass = '';

    if (layoutManager.tv) {
        cssClass += ' show-focus';    // space because it must be added to the base class

        if (enableFocusTransform) {
            cssClass += ' show-animation';
        }
    }

    // Here may be variations for other layouts

    return cssClass;
}
```

Reply:
I would just add the method as an export to CardBuilder for now, and place a TODO there that states this is a workaround until the component is more generic.

Reply:
I would be fine with you just adding `TODO`s for now